### PR TITLE
Bugfix/duplicate error messages

### PR DESCRIPTION
**Pull request by radireddy**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

## Title of the Pull Requests 
Duplicate error messages are shown on PCC general info page

### Description of the Change
The error message was added two to scope. I have removed one entry.

### Possible Drawbacks
none

### Applicable Issues
MANAGER-895

### CODE REVIEW

File: client/app/dedicatedCloud/security/dedicatedCloud-security.controller.js [hunk 1]
```diff
@@ -151,7 +151,7 @@ angular.module('App').controller('DedicatedCloudSecurityCtrl', function ($rootSc
   };
 
   self.onError = function (data) {
-    $scope.setMessage($translate.instant('dedicatedCloud_dashboard_loading_error'), data.data);
+    $scope.setMessage($translate.instant('dedicatedCloud_dashboard_loading_error'), _.get(data, 'data', data));
```
Comment (marie-j) [inline client/app/dedicatedCloud/security/dedicatedCloud-security.controller.js]: Maybe rename `data` param to `error` for instance  to prevent confusion

Comment (radireddy) [reply client/app/dedicatedCloud/security/dedicatedCloud-security.controller.js]: done. renamed data to error

**Comment by antleblanc** [comment]
You can squash your commits and get only this:

```txt
fix(dedicatedcloud): remove duplicate error messages
```

> use the imperative, present tense: "change" not "changed" nor "changes"

ref: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#subject

Thanks.

**Comment by antleblanc** [comment]
@radireddy you're right. I will use the "Squash and merge" feature to apply the commit suggestion I made.

**Pull request merged**